File: find/find.pl
use File::Find;
use FileHandle;

my $dir = $ARGV[0] || die "Please input subdirectory name";

my $find = sub {
	my $file = $_;
	if(-f $file) {
		my $fh = new FileHandle($file);
		local $/ = undef;
		my $con = <$fh>;
		$fh->close();

		print "Opened: ".$File::Find::name."\n";
		print "Content: ".$con;

	}
};
print "Find in ".$dir."\n";
find($find, $dir);

File: find/find.py
import os
import sys
argvs = sys.argv 
if len(sys.argv) > 1:
	dir = argvs[1]
else:
	sys.exit("Please specify subdirectory")

print("Find in " + dir)
for root, dirs, files in os.walk(dir):
    for file in files:
#        if file.endswith(".py"):
        print(os.path.join(root, file))
